### PR TITLE
c2wiDjMy Popup

### PR DESCRIPTION
**Pull request by YevgenChop**
- Added missing Text component
- Added Popup, Drawer, TinyPopup components
- Added stories
- Added useKeyUp and usePopup hooks

### CODE REVIEW

**Comment by YevgenChop** [comment]
[Popup](https://trello.com/c/c2wiDjMy/21-popup)

File: src/components/popup/drawer/drawer-section.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React, { FC } from 'react';
+import { Box, BoxProps } from 'rebass';
+
+// Components
+import Labeling from '../../typography/labeling';
+
+export interface DrawerSectionProps extends Omit<BoxProps, 'css' | 'title'> {
+  title: React.ReactNode;
```
Comment (giardiv) [inline src/components/popup/drawer/drawer-section.tsx:8]: Force a string

**Comment by giardiv** [comment]
Some UI feedbacks

- Drawer: the button should have a fixed position, it doesnt follow while we scroll within the drawer itself
- The component "Drawer section" should have a border top part of the component itself
- Enable leave the popup on clicking outside of it, like the default one

**Review by giardiv — CHANGES_REQUESTED** [review]

File: src/components/popup/drawer/drawer-section.tsx (hunk 1 above)
Comment (YevgenChop) [reply src/components/popup/drawer/drawer-section.tsx:8]: Sorry, I can't use it just as a string, because I put CLose button in the Drawer component.

**Comment by YevgenChop** [comment]
> Some UI feedbacks
> 
> * Drawer: the button should have a fixed position, it doesnt follow while we scroll within the drawer itself
> * The component "Drawer section" should have a border top part of the component itself
> * Enable leave the popup on clicking outside of it, like the default one

File: src/components/popup/drawer/drawer-section.tsx (hunk 1 above)
Comment (giardiv) [reply src/components/popup/drawer/drawer-section.tsx:8]: But drawer section does not has to be close, no?

**Review by giardiv — CHANGES_REQUESTED** [review]

**Pull request merged**